tutorials/sphinx-tutorials/export.py

```diff
@@ -0,0 +1,393 @@
+
+output = exported_policy.module()(pixels=pixels)
+print("Exported module output", output)
+
+#####################################
+# ``torch.export`` has many other features that we will explore further below. Before this, let us just do a small
+# digression on exploration and stochastic policies in the context of test-time inference.
+#
+# Working with stochastic policies
+# --------------------------------
+#
+# As you probably noted, above we used the :class:`~torchrl.envs.set_exploration_type` context manager to control
+# the behaviour of the policy. If the policy is stochastic (e.g., the policy outputs a distribution over the action
+# space like it is the case in PPO or other similar on-policy algorithms) or explorative (with an exploration module
+# appended like E-Greedy, additive gaussian or Ornstein-Uhlenbeck) we may want or not want to use that exploration
+# strategy in its exported version.
+# Fortunately, export utils can understand that context manager and as long as the exportation occurs within the right
+# context manager, the behaviour of the policy should match what is indicated. To demonstrate this, let us try with
+# another exploration type:
+
+with set_exploration_type("RANDOM"):
+    exported_stochastic_policy = torch.export.export(
+        policy_transform.select_out_keys("action"),
+        args=(),
+        kwargs={"pixels": pixels},
+        strict=False,
+    )
+
+#####################################
+# Our exported policy should now have a random module at the end of the call stack, unlike the previous version.
+# Indeed, the last three operations are: generate a random integer between 0 and 6, use a random mask and select
+# the network output or the random action based on the value in the mask.
+#
+print("Stochastic policy")
+exported_stochastic_policy.graph_module.print_readable()
+
+
+from tempfile import TemporaryDirectory
+
+#####################################
+# AOTInductor: Export your policy to pytorch-free C++ binaries
+# ------------------------------------------------------------
+#
+# AOTInductor is a PyTorch module that allows you to export your model (policy or other) to pytorch-free C++ binaries.
+# This is particularly useful when you need to deploy your model on devices or platforms where PyTorch is not available.
+#
+# Here's an example of how you can use AOTInductor to export your policy, inspired by the
+# `AOTI documentation <https://pytorch.org/docs/main/torch.compiler_aot_inductor.html>`_:
+#
+
+from torch._inductor.package import load_package
+
+with TemporaryDirectory() as tmpdir:
+    path = str(Path(tmpdir) / "model.pt2")
+    with torch.no_grad():
+        so_path = torch._inductor.aoti_compile_and_package(
+            exported_policy,
+            args=(),
+            kwargs={"pixels": pixels},
+            # Specify the generated shared library path
+            package_path=path,
+        )
+
+    compiled_module = load_package(str(Path(tmpdir) / "model.pt2"))
```

Comment:
You can also use `torch._inductor.aoti_load_package` which is a wrapper around `load_package`